ReplaceCharacter.java

```diff
@@ -0,0 +1,48 @@
+package com.company;
+
+import java.util.Scanner;
+
+import static com.company.Utils.println;
+
+public class ReplaceCharacter {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        println("Enter any word: ");
+        String word = input.nextLine();
+        println("Enter character you want to replace from word: ");
+        char oldchar = input.next().charAt(0);
+        println("Enter new character: ");
+        char newchar = input.next().charAt(0);
+        // replaceChar() method returns either null or a required string
+        if (replaceChar(word, oldchar, newchar) == null)
```

Comment:
you can store the result while calling the function. And then check result in if else.